pico/tools/etc/spotbugs/exclude.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<FindBugsFilter
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="https://github.com/spotbugs/filter/3.0.0"
+        xsi:schemaLocation="https://github.com/spotbugs/filter/3.0.0 https://raw.githubusercontent.com/spotbugs/spotbugs/3.1.0/spotbugs/etc/findbugsfilter.xsd">
+
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name="io.helidon.pico.tools.AbstractCreator"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name="io.helidon.pico.tools.AbstractFilerMessager$DirectFilerMessager"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name="io.helidon.pico.tools.AbstractFilerMessager$DirectFileObject"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name=" io.helidon.pico.tools.AbstractGeneralCreatorResponse$Builder"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name="io.helidon.pico.tools.CodeGenFiler"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name="io.helidon.pico.tools.DefaultApplicationCreator"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name="io.helidon.pico.tools.ModuleUtils"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+    <Match>
+        <!-- Path comes from config or code -->
+        <Class name="io.helidon.pico.tools.TypeTools"/>
+        <Bug pattern="PATH_TRAVERSAL_IN"/>
+    </Match>
+
+    <Match>
+        <Class name="io.helidon.pico.tools.DefaultApplicationCreator"/>
```

Comment:
There should be a clear comment explaining why the check should be excluded

Reply:
pushed